backend/index.js

```diff
@@ -1,12 +1,17 @@
 const express = require('express')
 const cors = require('cors')
 const router = require('./api/router')
+const path = require('path')
+
+require('dotenv').config();
```

Comment:
Also, since it will be a `devDependency`, let's make it optional.

```js
try {
  require('dotenv').config()
} catch (e) {
  console.warn('Cannot load dotenv:', e.message)
}